### ISSUE TITLE
Questions about mp3d datasets 

### ISSUE THREAD

**Issue by huangzizhen** [ggeorgak11/UPEN#4]
Thank you for your work.
Whether mp3d data set tests in the paper all use the starting point of each scenario provided by the official mp3d test set？

When I ran the pre-trained model you provided, I couldn't get the 141.6 of the map_accuracy_m2 metric mentioned in the paper, I got 122.92？

![image](https://github.com/ggeorgak11/UPEN/assets/27364814/9e0f195b-c640-4508-914c-674b790054ee)

Looking forward to your reply. @ggeorgak11
